ui/oopsyraidsy/data/06-ew/raid/p4s.ts

```diff
@@ -0,0 +1,61 @@
+import NetRegexes from '../../../../../resources/netregexes';
+import ZoneId from '../../../../../resources/zone_id';
+import { OopsyData } from '../../../../../types/data';
+import { OopsyTriggerSet } from '../../../../../types/oopsy';
+
+// TODO: Cursed Casting (pox) when you shouldn't (it hits but does no damage if correct)
+// TODO: Vengeful Belone (orbs) incorrect debuff
+// TODO: Director's Belone (towers) missed tower / incorrect tower?
+
+export type Data = OopsyData;
+
+const triggerSet: OopsyTriggerSet<Data> = {
+  zoneId: ZoneId.AsphodelosTheFourthCircleSavage,
+  damageWarn: {
+    'P4S Acid Pinax': '69D4', // standing on acid square
+    'P4S Lava Pinax': '69D5', // standing on fire square
+    'P4S Well Pinax': '69D6', // standing on water square
+    'P4S Levinstrike Pinax': '69D7', // standing on lightning square
+    'P4S Acid Pinax Elemental Belone': '69F5', // unsafe square during Elemental Belone
+    'P4S Lava Pinax Elemental Belone': '69F6', // unsafe square during Elemental Belone
+    'P4S Well Pinax Elemental Belone': '69F7', // unsafe square during Elemental Belone
+    'P4S Levinstrike Pinax Elemental Belone': '69F8', // unsafe square during Elemental Belone
+    'P4S Levinstrike Mekhane': '69D3', // lightning pinax ability
```

Comment:
Am I misunderstanding this one? It's totally possible to take non-lethal damage greater than zero on Levinstrike Pinax, particularly as a tank.

Reply:
Haha whoops, you're quite right.